cluster/gce/gci/configure-helper.sh

```diff
@@ -1575,6 +1613,9 @@ if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
   create-master-auth
   create-master-kubelet-auth
   create-master-etcd-auth
+  if [[ "${ENABLE_APISERVER_ADVANCED_AUDIT:-}" == "true" ]]; then
```

Comment:
Done (moved to where we setup the mount config for the policy file in `start-kube-apiserver`)